components/brave_rewards/resources/extension/brave_rewards/_locales/en_US/messages.json

```diff
@@ -298,7 +298,7 @@
     "description": "Error title when user missed a grant."
   },
   "unVerifiedCheck": {
-    "message": "Check Again...",
+    "message": "Refresh Status",
```

Comment:
Pls update line 51 to `Verified Creator` away from the term publisher.

Reply:
Done.